src/components/Modals/MembersSkillUpdateModal/Presentation.tsx

```diff
@@ -16,27 +24,102 @@ export default function MembersSkillUpdateModalPresentation({
   onClose: () => void;
   isOpen: boolean;
 }) {
+  const { username, picture, firstName, lastName } =
+    useSelector((state: RootState) => state.superUserOption);
+  const [isTagsOpen, setIsTagsOpen] = useState(false);
```

Comment:
please add better formatting 
you can club similar apis together and then leave a line before clubbing another apis together becomes more readable